recipes/nbappinator/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "nbappinator" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nbappinator-{{ version }}.tar.gz
+  sha256: 021ad8dc6a4f8c55562e95891dc67708e40fa639f85c9d60d6864f58d77aae14
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - ipywidgets >=7.8.0,<7.9.dev0
```

Is this intentionally pinned or just an "it works for me" situation? Upper bounds are usually disregarded, but not sure if there's anything special with ipywidgets. I do see [some pins with major constraints](https://github.com/search?type=code&q=org%3Aconda-forge+path%3Ameta.yaml+%22-+ipywidgets%22), but no with minor. Would you be opening to reconsider something like:

```suggestion
    - ipywidgets >=7.8.0,<8.0.dev0
```

---

The upper bound is an intentional workaround for some dependencies that don't play well together with newer versions of ipywidgets in certain vendored (managed) Jupyter environments.

Ipywidgets jumped from 7.8.x to 8.x: https://pypi.org/project/ipywidgets/#history

I'd prefer leaving at 7.8.x than changing: since there's no reason to expect a 7.9 and if one appears, we'd want to explicitly test with it.